### DISCUSSION TITLE
[v3] wire:loading does not work with @livewireScriptConfig

### DISCUSSION THREAD

**Discussion by coleshirley** [livewire/livewire#6460] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.0-beta.10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. manually load alpine and livewire with @livewireScriptConfig according to the docs
2. add an element with wire:loading
3. the element is always present on the screen
4. remove the manually loaded alpine and livewire and @livewireScriptConfig
5. the wire:loading element functions as expected

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
@coleshirley Have you got `@livewireStyles` in your app layout? `@livewireScriptConfig` is just for scripts.

**Reply by coleshirley** [reply]
I do not. I guess that wasn't really clear just from this section of the docs https://livewire.laravel.com/docs/installation#manually-bundling-livewire-and-alpine

Now that I'm looking for it I do see it mentioned further up the page. I'll probably put a PR together to make that a little more clear.

Thanks!
